modules/nw-ovn-kubernetes-rollback.adoc

```diff
@@ -8,7 +8,7 @@
 [id="nw-ovn-kubernetes-rollback_{context}"]
 = Using the offline migration method to roll back to the OpenShift SDN network plugin
 
-As a cluster administrator, you can roll back to the OpenShift SDN Container Network Interface (CNI) network plugin by using the offline migration method. During the migration you must manually reboot every node in your cluster. With the offline migration method, there is some downtime, during which your cluster is unreachable. 
+As a cluster administrator, you can roll back to the OpenShift SDN Container Network Interface (CNI) network plugin by using the offline migration method. During the migration you must manually reboot every node in your cluster. With the offline migration method, there is some downtime, during which your cluster is unreachable.
```

Comment:
Rewrite as: "Cluster administrators can roll back to the OpenShift..."

Reply:
:heavy_check_mark: